### ISSUE TITLE
Can't pull mirantis/kubeadm-dind-cluster:v1.9 on cluster up

### ISSUE THREAD

**Issue by stealthybox** [Mirantis/kubeadm-dind-cluster#67]
When starting a 1.9 cluster using the fixed version script we fail to pull the image from docker hub:
```
./fixed/dind-cluster-v1.9.sh up
* Making sure DIND image is up to date
Error response from daemon: manifest for mirantis/kubeadm-dind-cluster:v1.9 not found
```
The `v1.9` tag hasn't been pushed upstream yet: https://hub.docker.com/r/mirantis/kubeadm-dind-cluster/tags/

Solution is to tag and publish this image in the mirantis repo.

### Temp workarounds for others:
build your own image locally -- use the non-fixed script:
```bash
build/build-local.sh
DIND_IMAGE=mirantis/kubeadm-dind-cluster:local ./dind-cluster.sh up
```
use mine:
```bash
DIND_IMAGE="stealthybox/kubeadm-dind-cluster:v1.9" ./dind-cluster.sh up
```

**Comment by chlung** [comment]
@stealthybox Thanks for the solution, but I am getting the following error.  Do you know how to fix my issue?  I am running on Mac.  Thanks.

* Saving a copy of docker host's /lib/modules 
* Starting DIND container: kube-master
sed: can't read /etc/kubeadm.conf.tmpl: No such file or directory

**Comment by stealthybox** [comment]
@chlung I think that's related to a more recent PR that switches from using a heredoc to using envsubst on a file (#60)

Is it broken with my image?  
Did you try building a fresh image?  

I can't guess why this would break without digging deeper into it.

**Comment by chlung** [comment]
@stealthybox I tried to build locally, but I am getting a build error.

* Saving images to: save.tar.lz4
build/../build/funcs.sh: line 77: lz4: command not found

I am running on Mac.  So, I am not sure if this is the problem.  Thanks for your help.

**Comment by stealthybox** [comment]
@chlung yeah, that's the problem: `brew install lz4`
The README needs to be updated.

**Comment by chlung** [comment]
@stealthybox thanks a lot.  I am about to build now.

**Comment by lukaszo** [comment]
1.9 image is pushed now and deploying with `./fixed/dind-cluster-v1.9.sh up` works.
